docker/Dockerfile

```diff
@@ -307,18 +307,14 @@ COPY --from=csrc-build /workspace/dist /precompiled-wheels
 
 COPY . .
 
-ARG GIT_REPO_CHECK=0
-RUN --mount=type=bind,source=.git,target=.git \
-    if [ "$GIT_REPO_CHECK" != "0" ]; then bash tools/check_repo.sh ; fi
-
 ARG vllm_target_device="cuda"
 ENV VLLM_TARGET_DEVICE=${vllm_target_device}
 
 # Skip adding +precompiled suffix to version (preserves git-derived version)
 ENV VLLM_SKIP_PRECOMPILED_VERSION_SUFFIX=1
 
+ARG VLLM_VERSION_OVERRIDE="0.0.0+src.build"
```

Comment:
> Also built image will no longer have a git-derived version string. What version will vllm --version report with this change?

@amrmahdi

It will show what is filled in here.